pkgs/top-level/python-packages.nix

```diff
@@ -4552,7 +4552,7 @@ in modules // {
 
       mkdir -p $out/bin
       mkdir -p $out/lib/${python.libPrefix}/site-packages
-      cp -r server/* $out/lib/${python.libPrefix}/site-packages
+      cp -r server/* $out/lib/${python.libPrefix}/site-packages             #*/
```

Comment:
Putting these things in because emacs mode doesn't know about `''` strings and treats the `/*` as a comment start, so everything afterwards looks like a comment...